docs/content.zh/docs/try-flink/table_api.md

```diff
@@ -28,51 +28,49 @@ under the License.
 
 # 基于 Table API 实现实时报表
 
-Apache Flink offers a Table API as a unified, relational API for batch and stream processing, i.e., queries are executed with the same semantics on unbounded, real-time streams or bounded, batch data sets and produce the same results.
-The Table API in Flink is commonly used to ease the definition of data analytics, data pipelining, and ETL applications.
+Apache Flink 提供 Table API 作为批流统一的关系型 API，例如，相同语法的查询在无界的实时流数据或者有界的批数据集上执行会得到一致的结果。  
```

Comment:
Apache Flink 提供了 Table API 作为批流统一的关系型 API。也就是说，在无界的实时流数据或者有界的批数据集上进行查询具有相同的语义，得到的结果一致。

Maybe semantics should be translated to '语义'  instead of '语法'?

Reply:
At the first glance,I think the keypoints was "the same SQL on diffrent type of data will get the same results",so 'semantics' related with 'queryes' and translated to '语法' was just fine.
After read your comment,maybe " SQL always produce the same semantics  " is the right keypoints and I agree with your comment.

Reply:
Okay